packages/core/data-transfer/src/strapi/providers/remote-destination/index.ts

```diff
@@ -78,33 +101,48 @@ class RemoteStrapiDestinationProvider implements IDestinationProvider {
       return new ProviderTransferError('Unexpected error');
     }
 
+    this.stats[step] = { count: 0 };
+
     return null;
   }
 
   async #endStep<T extends Client.TransferPushStep>(step: T) {
     try {
-      await this.dispatcher?.dispatchTransferStep({ action: 'end', step });
+      const res = await this.dispatcher?.dispatchTransferStep<{
+        ok: boolean;
+        stats: { started: number; finished: number };
+      }>({
+        action: 'end',
+        step,
+      });
+
+      return { stats: res?.stats, error: null };
```

Comment:
To make it consistent with the rest of the possible response (goes from `Stats | null | undefined` to `Stats | null`)

```suggestion
      return { stats: res?.stats ?? null, error: null };
```